roles/installer/tasks/59_power_off_cluster_servers.yml

```diff
@@ -31,7 +58,9 @@
       retries: 10
       delay: 5
       with_items: "{{ groups['poweroff_hosts'] }}"
-      when: groups['poweroff_hosts'] is defined
+      when:
+        - groups['poweroff_hosts'] is defined
+        - item in installer_powered_on_hosts | default([])
```

Comment:
does it make sense to use loop_var here as well?

Reply:
sure, I replaced all the `with_items`  with loop in this file.